### ISSUE TITLE
radicale/httputils.py uses "importlib.abc.Traversable" which will be removed from Python 3.14

### ISSUE THREAD

**Issue by pbiering** [Kozea/Radicale#1428]
Found warning:

```
radicale/httputils.py:44: DeprecationWarning: 'importlib.abc.Traversable' is deprecated and slated for removal in Python 3.14
    _TRAVERSABLE_LIKE_TYPE = Union[importlib.abc.Traversable, pathlib.Path]
```

See also:
https://docs.python.org/3/library/importlib.resources.abc.html

As support for Python 3.9 can't be dropped for the next years because of LTS of Enterprise Linux (https://github.com/Kozea/Radicale/wiki), the code must support "old" and "new" in parallel in the future.

**Comment by pbiering** [comment]
duplicate of https://github.com/Kozea/Radicale/issues/1428

**Comment by The-Compiler** [comment]
This is #1428, you probably mean #1516?
